### ISSUE TITLE
nvim-cmp: cannot select first item on completion

### ISSUE THREAD

**Issue by mattdibi** [mattdibi/dotfiles#14]
When using `nvim-cmp` I cannot select the first item of the completion list.

https://user-images.githubusercontent.com/22748355/209482775-e42c72aa-1c8c-473a-a9e7-487f419d4d45.mov

**Comment by mattdibi** [comment]
Fixed in abca43c5c8d95a88bf558dea999d9cf6774f3cce

**Comment by mattdibi** [comment]
More correct answer: https://github.com/VonHeikemen/lsp-zero.nvim/blob/main/advance-usage.md#dont-preselect-first-match

```lua
local lsp = require('lsp-zero')
lsp.preset('recommended')

lsp.setup_nvim_cmp({
  preselect = 'none',
  completion = {
    completeopt = 'menu,menuone,noinsert,noselect'
  },
})

lsp.setup()
```
